source/server-discovery-and-monitoring/server-discovery-and-monitoring-logging-and-monitoring.rst

```diff
@@ -0,0 +1,713 @@
+.. role:: javascript(code)
+  :language: javascript
+
+=========================================
```

Comment:
I'm a little concerned about the way that it seems like GitHub is considering this a brand new file rather than a rename of the existing one since I think it is going to totally blow out the git history for this file.

it also makes this more difficult to review since you can't tell from looking at this file what is old and what is new.

can we hold off on the file rename for now and do that in a separate commit at the end? 

(I'll wait to review this file until it is renamed back since it will be easier then)

Reply:
Was this issue addressed? It still looks like a new file to me in the PR diff

Reply:
Yes it had the original naming for a while, and I renamed it back today.
LMK if I need to rename it temporary back.